repairnator/src/main/java/fr/inria/spirals/repairnator/process/BuildToBeInspected.java

```diff
@@ -0,0 +1,39 @@
+package fr.inria.spirals.repairnator.process;
+
+import fr.inria.spirals.jtravis.entities.Build;
+import fr.inria.spirals.jtravis.entities.BuildStatus;
+
+/**
+ * Created by fernanda on 24/02/17.
+ */
+public class BuildToBeInspected {
+
+    private Build build;
+    private Build previousBuild;
+    // For RepairNator, the status is about the build, and for Bears it is about the previous build
+    private BuildStatus status;
```

Comment:
We should not use the build status but use another status, for example `ScannedBuildStatus` which would contain: BUILD_AND_FAIL, BUILD_WITH_TEST_CHANGES, ONLY_FAIL.